external/GravitationalEffectiveSource/README.md

```diff
@@ -0,0 +1,9 @@
+This is a modified version of https://github.com/barrywardell/GravitationalEffectiveSource
```

Comment:
Mention that the repo is private, since people will generally expect a URL to work if given with no context.

Reply:
Ok I added some information to the readme (squashed in)